README.md

```diff
@@ -130,6 +130,7 @@ To learn more and get started:
 - Read the [documentation](https://aka.ms/sk/learn)
 - Learn how to [contribute](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md) to the project
 - Join the [Discord community](https://aka.ms/SKDiscord)
+- Attend [regular office hours and SK community events](https://aka.ms/sk-community-calendar)
```

Comment:
This link just immediately downloads an ics file, which is not super useful if I'm just curious about what the office hours are about. Can we have it link to a .md file in the docs/ that explains what/when the office hours are, and gives an explicit link to "download .ics file"

Reply:
Updated!